prime-dotnet-webapi/Controllers/HealthAuthoritiesController.cs

```diff
@@ -307,5 +315,63 @@ public async Task<ActionResult> GetSitesByCareType(int healthAuthorityId, int he
 
             return Ok(siteIds);
         }
+
+        // PUT: api/health-authorities/1/organization-agreement
+        /// <summary>
+        ///    Create or Update health auth organization agreement
+        /// </summary>
+        /// <param name="healthAuthorityId"></param>
+        /// <param name="documentGuid"></param>
+        [HttpPut("{healthAuthorityId}/organization-agreement", Name = nameof(CreateOrUpdateOgranizationAgreement))]
+        [Authorize(Roles = Roles.EditSite)]
```

Comment:
Should that be `Roles.PrimeMaintenance` instead?

Reply:
Forgot we had that role!